app/helpers/remote_execution_helper.rb

```diff
@@ -16,41 +16,50 @@ def host_counter(label, count)
   def template_invocation_status(task, parent_task)
     if task.nil?
       if parent_task.result == 'cancelled'
-        icon_text('warning-triangle-o', _('cancelled'), :kind => 'pficon')
+        'cancelled'
       else
-        icon_text('question', 'N/A', :kind => 'fa')
+        'N/A'
       end
     elsif task.state == 'running'
-      icon_text('running', _('running'), :kind => 'pficon')
+      'running'
     elsif task.state == 'planned'
-      icon_text('build', _('planned'), :kind => 'pficon')
+      'planned'
     else
       case task.result
         when 'warning', 'error'
-          icon_text('error-circle-o', _('failed'), :kind => 'pficon')
+          'error'
         when 'cancelled'
-          icon_text('warning-triangle-o', _('cancelled'), :kind => 'pficon')
+          'cancelled'
         when 'success'
-          icon_text('ok', _('success'), :kind => 'pficon')
+          'success'
         else
           task.result
```

Comment:
This could be simplified to "if result is warning return error, otherwise return the result". The part above could be simplified as well